.clj-kondo/config.edn

```diff
@@ -89,10 +88,7 @@
 
   :unresolved-var
   {:exclude
-   [colorize.core
-    metabase.test/with-gtaps
-    metabase.test/with-gtaps-for-user
-    metabase.test/with-user-attributes]}
+   [colorize.core]}
```

Comment:
I removed the enterprise-only stuff from `metabase.test`. It was too hard to debug when it failed to load. I moved it to a new namespace called `metabase-enterprise.test` and updated things to use that.